craft_providers/multipass/multipass.py

```diff
@@ -386,3 +387,65 @@ def umount(self, *, mount: str) -> None:
                 brief=f"Failed to unmount {mount!r}.",
                 details=errors.details_from_called_process_error(error),
             ) from error
+
+    def wait_until_ready(
+        self, *, retry_wait: float = 0.25, timeout: Optional[float] = None
+    ):
+        """Wait until Multipass is ready (upon install/startup).
+
+        :param retry_wait: Time to sleep between retries.
+        :param timeout: Optional time.time() stamp to timeout after.
```

Comment:
Specifying an amount of time seems more common than defining a deadline when setting timeout values.

Reply:
Fixed as well as improved deadline time check..

Reply:
Thanks. But it seems that the timeout parameter description in the docstring still describes a deadline?

Reply:
Indeed, forget to fix it here. Fixed!